src/Crate/PDO/PDOStatement.php

```diff
@@ -202,7 +202,10 @@ public function execute($input_parameters = null)
             if (is_int($parameter) && $zero_based) {
                 $parameter++;
             }
-            $this->bindValue($parameter, $value);
+            // to correctly save arrays and objects, we need to pass
```

Comment:
Seems more like an eloquent issue to me.

[This is how it should be done](https://github.com/doctrine/dbal/blob/9e7954694971a5fab6ebabb38f9ffeec49d0d2ad/lib/Doctrine/DBAL/Connection.php#L822-L824)